### PR TITLE
adding module load and other improvements

### PR DESCRIPTION
**Pull request by dringeis**
- Adding a section in README to install via symlinks
- removing unused code
- adding the loading of the modules and changin the order to times get updated even if we do not recompile
- fixing the qsub bug (replacing -q by -s)
- adding a symlink link to the run folder in the `decks` folder in which econ was launched.

### CODE REVIEW

File: econ [hunk 1]
```diff
@@ -354,14 +354,21 @@ function post_build() {
     log_info "cp -R ${er}/data/landice/config ${rundir}/decks/${runid}/"
     cp -R ${er}/data/landice/config ${rundir}/decks/${runid}/
   fi
+  
+  echo ${STARTDIR}/${runid}
```
Comment (mankoff) [inline econ]: Please use `log_info` or `log_warn`, not `echo`

File: README.org [hunk 2]
```diff
@@ -18,6 +18,15 @@ wget https://github.com/NASA-GISS/econ/archive/refs/heads/main.zip -O econ.zip
 unzip -j -o econ.zip econ-main/econ econ-main/econ_includes.sh
 #+END_SRC
 
+** Installation with git access
```
Comment (mankoff) [inline README.org:21]: Good, although it doesn't need to be in `exec` folder. You need to run it from `decks`, but from `decks` can call `/path/to/econ.git/econ`.

File: econ (hunk 1 above)
Comment (dringeis) [reply econ]: ahah yes, forgot to remove that line

Comment (dringeis) [inline econ]:
```suggestion
```

File: README.org (hunk 2 above)
Comment (dringeis) [reply README.org:21]: True, but I thought it would be more logical to keep it in exec. As you wish, we could remove this section.

Comment (mankoff) [reply README.org:21]: I'm happy to keep it - just an FYI for your development purposes.

**Review by mankoff — APPROVED** [review]

**Pull request merged**